### DISCUSSION TITLE
Can't see any Releases with the feature preview enabled 🤔

### DISCUSSION THREAD

**Discussion by filippo-orru** [community/community#6362] [Other Feature Feedback, Questions, & Ideas] [answered]
Simple as that. With the new feature enabled, it says "no releases yet" and without it, I can see them just like I used to.

**Comment by MylesBorins** [comment, marked as answer]
@ffactory-ofcl is it possible that you only have tags but no GitHub releases? Part of this work was making the difference between the two more explicit and separating releases from tags

edit: marking as answered due to lack of response. I'm 99% sure that the reason why is due to tags not having explicit releases, please let me know if I am incorrect

**Reply by filippo-orru** [reply]
You're right. We'll probably stick to tags as the only benefit I see in releases is the option to add binaries, which we don't need. Thanks for the clarification.
